src/main/java/emissary/output/DropOffPlace.java

```diff
@@ -245,9 +246,16 @@ public List<IBaseDataObject> agentProcessHeavyDuty(final List<IBaseDataObject> p
                 objectMetricsLog.info(appendEntries(outputObjectMetrics(tld, objectMetricsFields)), "Finished DropOff");
             }
 
-            logger.info("Finished DropOff for object {}, with external id: {}, with total processing time: {}ms, with filetype: {}",
-                    tld.getInternalId(), this.dropOffUtil.getBestId(tld, tld), (new Date().getTime() - tld.getCreationTimestamp().getTime()),
-                    tld.getFileType());
+            if (outputCompletionPayloadSize) {
+                logger.info("Finished DropOff for object {}, with external id: {}, with total processing time: {}ms, with filetype: {}, payload size: {}",
```

Comment:
Is it common for us to specify a size without units in log messages? (legitimate question, not trolling)
```suggestion
                logger.info("Finished DropOff for object {}, with external id: {}, with total processing time: {}ms, with filetype: {}, payload size: {} bytes",
```

Reply:
I think we generally do. But don't think we have a hard set rule. Definitely good practice. Especially since the time before specifies (ms).

Just an oversight on my part.